wgpu-hal/src/gles/egl.rs

```diff
@@ -1073,13 +1100,15 @@ impl super::Adapter {
     /// - The underlying OpenGL ES context must be current.
     /// - The underlying OpenGL ES context must be current when interfacing with any objects returned by
     ///   wgpu-hal from this adapter.
+    /// - The underlying OpenGL ES context must be current when dropping this adapter and when
+    ///   dropping any objects returned from this adapter.
```

Comment:
This new point might be covered by "interfacing"? I want to make sure it was clear though.

Reply:
More docs > less docs :)